boards/riscv/beagleV/doc/index.rst

```diff
@@ -0,0 +1,71 @@
+.. _beagelV:
```

Comment:
Typo

Reply:
Fixed in e7ce67a4ca44281ea4d39cb4a7449b113738b8f3